site/content/docs/latest/tutorials/managing-carvel-packages.md

```diff
@@ -201,46 +198,46 @@ The reason why is that kapp-controller forces to explicitly provide needed privi
+At this moment, kapp-controller performs the required actions to start creating the Kubernetes resources defined by the package. This process is known as _reconciliation_.
 
 ### Viewing the Installed Packages
 
 Viewing the installed Carvel Packages in Kubeapps is the same experience as viewing any other installed package (such as a Helm Chart) in Kubeapps.
 
 > **TIP**: Please refer to the [user documentation](../howto/dashboard.md) for more information on how to use Kubeapps as a user.
 
-Go to the `Applications` tab to see every Application that has been installed in the cluster. Click on _show apps in all namespaces_ to view the ones currently installed in every namespace of the cluster.
+Go to the `Applications` tab to see every Application installed in the cluster. Click on _show apps in all namespaces_ to view the ones currently installed in every namespace of the cluster.
 
-The following example shows an example of the Applications page with two Carvel Packages installed:
+The following image shows an example of the Applications page with two Carvel Packages installed:
 
 ![Installed applications page](../img/carvel-apps.png)
 
-Since the reconciliation process can eventually fail for several reasons, this page will show the current status of each application. If it is `deployed` it means that the application is successfully running in the cluster.
+Since the reconciliation process can eventually fail for several reasons, this page shows the current status of each application. If it is `deployed` it means that the application is successfully running in the cluster.
 
 Next, click on the application you want to view, for example, `my-test` to go to the details page, as depicted in the following image:
 
 ![Details page of an installed Carvel Package](../img/carvel-details.png)
 
-As in any other packaging format, this page will display those Kubernetes resources that have been created as a result of the Package installation.
+As in any other packaging format, this page displays those Kubernetes resources that have been created as a result of the Package installation.
 Besides, the current values are shown at the end of the page.
 
-Next, you can click on the `Delete` button to uninstall the application or the `Upgrade` button to edit the values of the application or update it to another version.
+Next, you can click on the **Delete** button to uninstall the application or the **Upgrade** button to edit the values of the application or update it to another version.
 
-> **NOTE**: as opposed to Helm Charts, Carvel Packages cannot be rolled back, hence there is no `Rollback` button.
+> **NOTE**: as opposed to Helm Charts managed by using Helm, Carvel Packages cannot be rolled back, hence there is no **Rollback** button.
 
 Finally, note that every Carvel Package installed through Kubeapps can also be managed by the [kapp](https://carvel.dev/kapp/) CLI using the `kapp inspect -a <APPLICATION_NAME>` command. For example:
```

Comment:
Here we are saying that stuff installed in Kubeapps is shown with kapp CLI.
Shouldn't we also mention the other way around? i.e. that packages installed from kapp CLI are also presented in Kubeapps.

Reply:
Agree